lib/foodcritic/command_line.rb

```diff
@@ -83,6 +107,68 @@ def initialize(args)
           e.recover args
         end
       end
+
+      [:cookbook_paths, :role_paths, :environment_paths, :include_rules, :exclude_paths].each do | pth|
+        @options[pth].map! { |c| File.expand_path(c, '.') } if @options[pth] && @options[pth] != [nil]
+        @options[pth].flatten!
+        @options[pth].uniq!
+      end
+    end
+
+    # Search for and load config files
+    def get_config(cbs)
+      require 'app_conf'
+      files_to_load = []
+      files_to_load << File.expand_path('~/.chef/foodcritic.yml', '.') if File.file?(File.expand_path('~/.chef/foodcritic.yml'))
+      ['.', cbs].flatten!.each do |mypth|
```

Comment:
Not my intention, per se, but to start in both the current directory & the specified cookbook directory and search upwards for any ".chef/foodcritic.yml" files.

Yes, technically that would allow you to put ".chef/foodcritic.yml" into each cookbook.